app/assets/stylesheets/application.scss

```diff
@@ -3,4 +3,10 @@
 @import "bootstrap";
 @import "bootstrap_overrides";
 @import "font-awesome-sprockets";
-@import "font-awesome";
\ No newline at end of file
+@import "font-awesome";
+
+.form-container {
+    max-width: 500px;
+    padding: 15px;
+    margin: 0 auto;
+}
```

Comment:
It's usually best practice not to include actual styles in your manifest file; I'd recommend abstracting this into a `forms.scss` file or something and including it here.